### PR TITLE
Add base Celery configuration, ready for projects to use

### PR DESCRIPTION
**Pull request by danielsamuels**

### CODE REVIEW

**Comment by dan-gamble** [comment]
A small README on what things would benefit from Celery would be 👍

**Comment by danielsamuels** [comment]
Writing it as we speak!

**Comment by danielsamuels** [comment]
Documentation is now available in the PT [here](https://github.com/onespacemedia/project-template/tree/feature/celery/%7B%7Bcookiecutter.repo_name%7D%7D), and will become the base README.md file in generated projects.

File: {{cookiecutter.repo_name}}/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+* `DJANGO_SETTINGS_MODULE={{cookiecutter.package_name}}.settings.local celery -A {{cookiecutter.package_name}} worker -E --loglevel=info`
+
+#### In production
+
+This will eventually be handled by server-manangement, but for now if you need it then use this as a guide:
+
+* `sudo add-apt-repository ppa:chris-lea/redis-server`
+* `sudo apt-get update`
+* `sudo apt-get install redis-server`
+* `ln -s /var/www/{{cookiecutter.package_name}}/config/celery.conf /etc/supervisor/conf.d/celery.conf`
```
Comment (lewiscollard) [inline {{cookiecutter.repo_name}}/README.md]: +sudo here?

**Review by lewiscollard — APPROVED** [review]

**Pull request merged**